python/src/trezorlib/messages.py

```
@@ -10659,6 +10659,19 @@ def __init__(
 
 class WebAuthnListResidentCredentials(protobuf.MessageType):
     MESSAGE_WIRE_TYPE = 800
+    FIELDS = {
+        100: protobuf.Field("index", "uint32", repeated=False, required=False, default=None),
+        101: protobuf.Field("request_list_index", "bool", repeated=False, required=False, default=None),
+    }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)